vassal-app/src/main/java/VASSAL/build/module/GameState.java

```diff
@@ -372,10 +385,51 @@ public boolean isGameStarted() {
    * sent to {@link GameModule#decode} and translated into a
    * {@link Command}, which is then executed.  The command read from the
    * file should be that returned by {@link #getRestoreCommand}.
+   *
+   * This version for binary compatibility w/ old style.
    */
   public void loadGame() {
+    loadGame(true);
+  }
+
+  /**
+   * Read the game from a savefile.  The contents of the file is
+   * sent to {@link GameModule#decode} and translated into a
+   * {@link Command}, which is then executed.  The command read from the
+   * file should be that returned by {@link #getRestoreCommand}.
+   *
+   * @param continuation if true then do the "old-style" version of load continuation
+   */
+  public void loadGame(boolean continuation) {
     final GameModule g = GameModule.getGameModule();
 
+    if (gameStarted && continuation) {
+      if (GlobalOptions.getInstance().isWarnOldContinuation()) {
+        final Object[] options = {
+          Resources.getString("GameState.anyway"),
+          Resources.getString("GameState.cancel"),
+          Resources.getString("GameState.dont_prompt_again")
+        };
+
+        final int result = JOptionPane.showOptionDialog(
+          g.getPlayerWindow(),
+          Resources.getString("GameState.old_continuation_warning"),
+          "",
+          JOptionPane.YES_NO_CANCEL_OPTION,
+          JOptionPane.QUESTION_MESSAGE,
+          null,
+          options,
+          options[0]
+        );
+        if (result == JOptionPane.NO_OPTION) {
+          return;
+        }
+        else if (result == 2) { // Turn Preference Off
```

Comment:
Can we use one of the predefined constants here? It's not obvious what `2` is.

Reply:
I imagine we can. "2" is the condition I found this in but I'll go try to find something appropriate that's equal to 2.